ApexOSA/filter.py

```
@@ -0,0 +1,271 @@
+            - State = False : the filter output is not enabled
+            - State = True : the filter output is enabled
+        """
+
+        if not self.__Simulation:
+            Command = "FILOUTENABLE?\n"
+            Send(self.__Connexion, Command)
+            State = Receive(self.__Connexion, 64)[:-1]
+            try:
+                State = bool(int(State))
+            except:
+                pass
```

Ruff (0.12.2)

Use `contextlib.suppress(BaseException)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(BaseException)`

(SIM105)